packages/mobile/src/components/core/UserGeneratedText.tsx

```diff
@@ -105,15 +150,15 @@ export const UserGeneratedText = (props: UserGeneratedTextProps) => {
 
   const renderLink = useCallback(
     (text: string, match: Match) => (
-      <TextLink
+      <Link
```

Comment:
might wanna double-check with dylan about this, i always forget which Link component is for which purpose.

Reply:
this `Link` is defined in this file :)  it uses `TextLink` under the hood